packages/types/src/codec/Compact.ts

```diff
@@ -147,6 +147,6 @@ export default class Compact extends Base<UInt | Moment> {
   }
 
   toU8a (isBare?: boolean): Uint8Array {
-    return Compact.encodeU8a(this.raw.toBn(), this.bitLength);
+    return Compact.encodeU8a(this.raw.toBn(), this.bitLength());
```

Comment:
Ahh... ok, to my question above, here it is used. (I should have probably left a note in there somewhere as to why we have `bitLength`)